src/views/virtualmachines/details/components/details/utils/operatingSystemHelper.ts

```diff
@@ -0,0 +1,31 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+import { NAME_OS_TEMPLATE_ANNOTATION, OS_TEMPLATE_LABEL } from '../../../../utils/constants';
+
+type StringHashMap = {
+  [key: string]: string;
+};
+
+const getPrefixedKey = (obj: StringHashMap, keyPrefix: string) =>
```

Comment:
Do you know what kind of map to expect? If so, why not name this something more descriptive?

Reply:
changed to `LabelsOrAnnotationsMap`, if you have a better suggestion, please let me know :)